src/base/QXmppConstants_p.h

```diff
@@ -1,4 +1,5 @@
 // SPDX-FileCopyrightText: 2016 Manjeet Dahiya <manjeetdahiya@gmail.com>
+// SPDX-FileCopyrightText: 2024 Filipe Azevedo <pasnox@gmail.com>
```

Comment:
addition of data / minor changes or changes without creative work don't need extra copyright lines